microsoft-edge/devtools-guide-chromium/whats-new/whats-new.md

```diff
@@ -23,22 +23,4 @@ ms.date: 05/16/2022
 * [What's New in DevTools (Microsoft Edge 102)](2022/05/devtools-102.md)
```

Comment:
change line 10 from:
```
<!-- add new pages to toc too -->
```
to:
```
<!-- maintenance notes:
* add the new page to toc.yml
* move eleventh oldest link into whats-new-archive.md
-->
```

Reply:
fixed

Reply:
fix not appearing, re-entering a comment